datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -4,6 +4,8 @@
 import re
 from contextlib import contextmanager
 
+from datadog_checks.dev.ssh_tunnel import find_free_port, get_ip
```

Comment:
Since it's reused, can we move `find_free_port` and `get_ip` to a more generic location ? 
e.g. `datadog_checks/dev/utils.py`